packages/vite/src/node/server/index.ts

```diff
@@ -206,6 +208,14 @@ export interface ViteDevServer {
     options?: EsbuildTransformOptions,
     inMap?: object
   ): Promise<ESBuildTransformResult>
+  /**
+   * Transform module code into SSR format.
```

Comment:
@antfu I think we should mark this as experimental so we can merge it and release it as part of vite 2.7
```suggestion
   * Transform module code into SSR format.
   * @experimental
```